plugin-catalog/src/components/plugins/Detail.tsx

```diff
@@ -233,11 +236,8 @@ export function PurePluginDetail({
                 ) : null
               ) : (
                 <>
-                  <LoadingButton progress={currentActionProgress} onCancel={onCancel} />
                   <Tooltip title={`${currentActionState}:${currentActionMessage}`}>
-                    <Typography variant="body1" sx={{ marginLeft: 1 }}>
-                      {currentAction}
-                    </Typography>
+                    <LoadingButton progress={currentActionProgress} onCancel={onCancel} />
```

Comment:
I'm not sure about this one. (not sure how the change looks, and haven't checked yet)